testing/src/main/java/io/grpc/testing/DeadlineSubject.java

```diff
@@ -102,32 +101,6 @@ private TolerantDeadlineComparison() {}
      * subject, and tolerance are all specified earlier in the fluent call chain.
      */
     public abstract void of(Deadline expectedDeadline);
-
-    /**
-     * Do not call this method.
-     *
-     * @throws UnsupportedOperationException always
-     * @deprecated {@link Object#equals(Object)} is not supported on TolerantDeadlineComparison
-     *     If you meant to compare deadlines, use {@link #of(Deadline)} instead.
-     */
-    @Deprecated
```

Comment:
This is actually here to scare people away from it.  I think the deprecated is present to make IDEs catch on more quickly that they shouldn't be using it.  The exception is to enforce it.  Truth makes it easy to accidentally call:


assertAbout(DeadlineSubject.deadline()).that(myDeadline).equals(otherDeadline);

vs.

assertAbout(DeadlineSubject.deadline()).that(myDeadline).IsEqual(otherDeadline);

The first asserts that the DeadlineSubject itself is equal  to the deadline, where as the latter compares the subject to the object.